include/lf.h

```diff
@@ -78,19 +78,19 @@ typedef struct {
   lf_pinbox_free_func *free_func;
   void *free_func_arg;
   uint free_ptr_offset;
-  std::atomic<uint32> pinstack_top_ver;    /* this is a versioned pointer */
-  std::atomic<uint32> pins_in_array;       /* number of elements in array */
+  std::atomic<uint64> pinstack_top_ver;    /* this is a versioned pointer */
+  std::atomicuint64> pins_in_array;        /* number of elements in array */
```

Comment:
Compilation-breaking typo

Reply:
Uhh, sorry about that, I thought I compiled my last version of this before pushing :(